ghcide/src/Development/IDE/Core/Rules.hs

```diff
@@ -211,10 +211,12 @@ getParsedModuleRule :: Rules ()
 getParsedModuleRule =
   -- this rule does not have early cutoff since all its dependencies already have it
   define $ \GetParsedModule file -> do
-    ModSummaryResult{msrModSummary = ms} <- use_ GetModSummary file
+    ModSummaryResult{msrModSummary = ms'} <- use_ GetModSummary file
     sess <- use_ GhcSession file
     let hsc = hscEnv sess
     opt <- getIdeOptions
+    modify_dflags <- getModifyDynFlags id dynFlagsModifyParser
+    let ms = ms' { ms_hspp_opts = modify_dflags $ ms_hspp_opts ms' }
```

Comment:
Isn't this `DynFlags` already modified in the `GetModSummary` rule?

Reply:
It's already modified with the `dynFlagsModifyGlobal`, but not the `*ModifyParser`

Reply:
Can we do away with the global one? Why do we need the parser variant?

Reply:
NVM, I just read the description